vendor/google.golang.org/protobuf/types/descriptorpb/descriptor.pb.go

```diff
@@ -42,10 +42,11 @@
 package descriptorpb
 
```

Comment:
This file should not be checked by lint.

Reply:
reverted